lang/cs/Org.Apache.REEF.Network/Group/Operators/Impl/BroadcastReceiver.cs

```diff
@@ -64,7 +64,7 @@ public class BroadcastReceiver<T> : IBroadcastReceiver<T>
             PipelineDataConverter = dataConverter;
             _topology = topology;
 
-            var msgHandler = Observer.Create<GroupCommunicationMessage>(message => _topology.OnNext(message));
+            var msgHandler = Observer.Create<GroupCommunicationMessage>(message => topology.OnNext(message));
```

Comment:
because it should not inherit from IObserver. Should it?

On Sun, Jun 7, 2015 at 11:42 PM, Julia Wang <notifications@github.com>
wrote:

> In
> lang/cs/Org.Apache.REEF.Network/Group/Operators/Impl/BroadcastReceiver.cs
> <https://github.com/apache/incubator-reef/pull/207#discussion_r31889554>:
>
> > @@ -64,7 +64,7 @@ public class BroadcastReceiver<T> : IBroadcastReceiver<T>
> >              PipelineDataConverter = dataConverter;
> >              _topology = topology;
> >
> > -            var msgHandler = Observer.Create<GroupCommunicationMessage>(message => _topology.OnNext(message));
> > +            var msgHandler = Observer.Create<GroupCommunicationMessage>(message => topology.OnNext(message));
>
> Why not use IOperatorTopology in operator constructors?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/incubator-reef/pull/207/files#r31889554>.
>